dashboard/app/controllers/registrations_controller.rb

```diff
@@ -173,7 +173,13 @@ def create
       end
 
       if ActiveModel::Type::Boolean.new.cast(params[:new_sign_up])
-        @user = Services::PartialRegistration::UserBuilder.call(request: request)
+        begin
+          @user = Services::PartialRegistration::UserBuilder.call(request: request)
+        rescue ActiveRecord::RecordInvalid => exception
+          return render json: {
+            error: exception
+          }, status: :bad_request
+        end
```

Comment:
If there's a different error, will this still return a 500?

Reply:
Yep! The third case in the description above is where I threw a 500 error, and it rendered the generic message shown if it's not a 400 error.

Reply:
🤦‍♀️ should've read the entire description. Sorry!

Reply:
No worries!